.github/workflows/buildtest.yml

```diff
@@ -9,15 +9,15 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
+        go-version: [1.16.x, 1.17.x, 1.18.x]
```

Comment:
shall we test build against 1.17.x and 1.18.x like other projects ?

Reply:
yep I miss that